src/services/statistics.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Injectable, BadRequestException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Appointment } from '../entities/index';
+import { ResponseStatisticsDto } from '../dtos';
+import { plainToClass } from 'class-transformer';
+
+@Injectable()
+export default class StatisticsService {
+  constructor(
+    @InjectRepository(Appointment)
+    private readonly appointmentRepository: Repository<Appointment>,
+  ) {}
+
+  async getStatistics(startDate: string, endDate: string, treatment?: string) {
+    if (!startDate || !endDate) {
+      throw new BadRequestException(
+        'Las fechas de inicio y fin son obligatorias.',
```

Comment:
💭  shouldn't this be in english?, because it's an error for the front end, not the user. Am I right?